components/support/migration/src/main/java/mozilla/components/support/migration/FennecMigrator.kt

```diff
@@ -384,4 +412,18 @@ class FennecMigrator private constructor(
             }
         }
     }
+
+    @SuppressWarnings("TooGenericExceptionCaught")
+    private fun migrateGecko(): Result<Unit> {
+        if (profile == null) {
+            crashReporter.submitCaughtException(IllegalStateException("Missing Profile path"))
+            return Result.Failure(IllegalStateException("Missing Profile path"))
+        }
+
+        return try {
+            GeckoMigration.migrate(profile.path)
```

Comment:
In every other migration we have debug log printouts before starting, after finishing migraton. Let's add them here as well.